src/Sulu/Bundle/HttpCacheBundle/EventSubscriber/TagsSubscriber.php

```diff
@@ -14,14 +14,16 @@
 use FOS\HttpCacheBundle\Http\SymfonyResponseTagger;
 use Sulu\Bundle\WebsiteBundle\ReferenceStore\ReferenceStoreInterface;
 use Sulu\Bundle\WebsiteBundle\ReferenceStore\ReferenceStorePoolInterface;
-use Sulu\Component\Content\Compat\StructureInterface;
+use Sulu\Content\Domain\Model\DimensionContentInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpKernel\KernelEvents;
 use Symfony\Component\Uid\Uuid;
 
 /**
- * Adds tags to the Symfony response tagger from all available reference stores.
+ * @internal This class should not be extended or initialized by any application outside of sulu.
+ *           You can create your own response listener to change the behaviour or use Symfony
+ *           dependency injection container to replace this service.
  */
 class TagsSubscriber implements EventSubscriberInterface
```

Comment:
@Prokyonn we should think of moving this to the content bundle as the listener more depends on `ConentController` attributes structure and dynamic content controller.